src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = {new Lottery().getRandomBall(),
+                        new Lottery().getRandomBall(),
+                        new Lottery().getRandomBall()};
```

Comment:
let's create a lottery variable one time and call `getRandomBall()` using it. Also, you can move the creation of balls in for loop that you wrote down below but in this case, you will need to use fori loop, not foreach)